.travis.yml

```diff
@@ -6,8 +6,6 @@ env:
     - GRADLE_OPTS="-Xmx512m"
 jdk:
   - oraclejdk8
-  - oraclejdk7
-  - openjdk7
```

Comment:
I'd prefer not to remove Java 7 builds while we still have Java 7 as our minimum required version of java.